yt/visualization/volume_rendering/off_axis_projection.py

```diff
@@ -483,6 +485,10 @@ def temp_weightfield(field, data):
             Nx=resolution[0],
             Ny=resolution[1],
         )
+        # Note: since dx was divided by wmax, we need to rescale by it
+        projected_weighted_qty *= wmax.d / np.sqrt(3)
+        projected_weight *= wmax.d / np.sqrt(3)
```

Comment:
ah! that explains the mostly constant factor that values were off by! nice catch.